data.json

```diff
@@ -1288,11 +1288,17 @@
     "githubUsername": "ahamedbasha-n",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/130546663-9ff92a77-14a1-4004-8d9c-dbc67fb027df.png",
     "issueNumber": 663
-  }, 
+  },
   {
     "name": "Anas Khan",
     "githubUsername": "anaskhan28",
     "imageUrl": "https://user-images.githubusercontent.com/87796038/130655483-7fd4dc84-9f0d-412b-acac-68ab178aaf3c.png",
     "issueNumber": 666
+  },
+  {
+    "name": "Mohammad Shakib",
+    "githubUsername": "mo-shakib",
```

Comment:
```suggestion
    "githubUsername": "Mo-Shakib",
```

Reply:
my bad! actually, it's my very first time that creating PR :)